src/components/ui/Box.tsx

```diff
@@ -132,18 +183,43 @@ export const Box: React.FC<BoxProps> = ({
           />
         )}
 
+        {/* Count label */}
         {!locked && !hideCount && !!count && count.greaterThan(0) && (
-          <Label
-            className={classNames(
-              "absolute -top-4 -right-3 px-0.5 text-xs z-10",
-              {
-                "z-20": isHover,
-              }
-            )}
+          <div
+            ref={labelRef}
+            className={classNames("absolute z-10", {
+              "z-20": isHover,
+            })}
+            style={{
+              right: `${LABEL_RIGHT_SIHFT_PX}px`,
+              top: `${LABEL_TOP_SIHFT_PX}px`,
+              pointerEvents: "none",
+            }}
           >
-            {isHover ? count.toString() : shortCount}
-          </Label>
+            <Label className="px-0.5 text-xxs">
+              {isHover && !showLabelChecker ? count.toString() : shortCount}
+            </Label>
+          </div>
         )}
+
+        {/* Transparent long count label to adjust the count label position on hover */}
+        {!locked &&
```

Comment:
Could we maybe store this long `&&` statement in a var with a clear name to sum up the condition?

Reply:
Addressed